src/util/date-input-helper.ts

```diff
@@ -1,34 +1,128 @@
 import dayjs from "dayjs";
+import isBetween from "dayjs/plugin/isBetween";
+import { INVALID_VALUE } from "../date-input/stand-alone-input";
+import { ChangeValueTypes, RawInputValues } from "../date-input";
+
+dayjs.extend(isBetween);
 
 export namespace DateInputHelper {
-    export const validate = (beginDate: string, afterDate: string): boolean => {
+    export const validate = (
+        beginDate: string,
+        afterDate: string,
+        disabledDates?: string[] | undefined,
+        between?: string[] | undefined
+    ): boolean => {
         let pass = false;
 
-        if (!beginDate.length || !afterDate.length) return false;
+        if (!beginDate?.length || !afterDate?.length) return false;
 
-        if (dayjs(beginDate).isAfter(afterDate)) {
-            pass = false;
+        if (dayjs(beginDate).isBefore(afterDate)) {
+            pass = true;
+        }
 
-            return pass;
+        if (dayjs(afterDate).isAfter(beginDate)) {
```

Comment:
unnecessary check. the first condition would have already covered the check. Furthermore both resolves to `pass = true`